e2e/pages/events/draftCaseManagementOrderEvent.page.js

```diff
@@ -46,6 +53,18 @@ module.exports = {
     I.fillField(this.fields.schedule.partiesPositions, schedule.partiesPositions);
   },
 
+  async enterDirection(direction) {
+    await I.addAnotherElementToCollection();
+    await draftDirections.enterTitleAndDescription('respondentDirectionsCustom', direction);
```

Comment:
Methods in `draftDirections` are not async so no need to await for a result. Remove unnecessary keyword. It is a shame that JS is so forgiving.